### PR TITLE
navbar

### PR DESCRIPTION
**Pull request by KcPele**
## Description
updated the navbar and made it more responsive
_Concise description of proposed changes, We recommend using screenshots and videos for better description_

## Additional Information
before
Screenshot from 2023-09-13 07-40-57
After
Screenshot from 2023-09-24 14-35-57

- [ ] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [ ] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #{issue number}_

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address:

### CODE REVIEW

**Comment by ZakGriffith** [comment]
Sorry, I'm not exactly sure what is updated here.  Can you provide a quick write up of the changes, or possibly screenshots/gif?

**Comment by damianmarti** [comment]
@KcPele reviewing the code seems that you are adding the connect button and the switch theme button to the header. Maybe we can add this information to the PR description and add some screenshots like @ZakGriffith suggested. The code looks good! Thanks!!

File: packages/nextjs/components/Header.tsx [hunk 1]
```diff
@@ -42,15 +42,21 @@ export const Header = () => {
       <li>
         <NavLink href="/projects">Projects</NavLink>
       </li>
-      <li>
+      <li className="dark:text-white">
```
Comment (damianmarti) [inline packages/nextjs/components/Header.tsx:45]: I'm not exactly a Tailwind expert, but I think that is better to use a text color class name that is set to another color on each theme at https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/blob/main/packages/nextjs/tailwind.config.js

**Comment by escottalexander** [comment]
image
The search bar should be aligned vertically at the same place the buttons to the right.

**Review by escottalexander — APPROVED** [review]
Looks good. Thanks for correcting the small issues.

**Pull request merged**